internal/sshserver/serverImpl.go

```diff
@@ -84,7 +94,13 @@ func (s *serverImpl) RunWithLifecycle(lifecycle service.Lifecycle) error {
 			break
 		}
 		s.wg.Add(1)
-		go s.handleConnection(tcpConn)
+		logger := s.logger
+		if useProxy {
+			proxyConn := tcpConn.(*proxyproto.Conn)
+			proxyIp := proxyConn.Raw().RemoteAddr()
+			logger = logger.WithLabel("fromProxy", proxyIp.(*net.TCPAddr).IP.String())
```

Comment:
This information needs to be added to the metadata so it can be logged in the audit log.

Reply:
Done